File: Rust/rui/todo/src/theme.rs
use vger::color::Color;

fn srgb_to_linear(s: f32) -> f32 {
    if s <= 0.04045 {
        s / 12.92
    } else {
        ((s + 0.055) / 1.055).powf(2.4)
    }
}

fn hex_linear(hex: &str) -> Color {
    let c = Color::hex(hex).unwrap();
    Color::new(
        srgb_to_linear(c.r),
        srgb_to_linear(c.g),
        srgb_to_linear(c.b),
        c.a,
    )
}

pub fn c_bg() -> Color { hex_linear("#1C1C1C") }
pub fn c_surface() -> Color { hex_linear("#2A2A2A") }
pub fn c_input() -> Color { hex_linear("#303030") }
pub fn c_text() -> Color { hex_linear("#D2D2D2") }
pub fn c_white() -> Color { hex_linear("#FFFFFF") }
pub fn c_muted() -> Color { hex_linear("#9B9B9B") }
pub fn c_accent() -> Color { hex_linear("#5DC2AF") }
pub fn c_cb_border() -> Color { hex_linear("#D9D9D9") }

File: Go/Go-GTK/theme.go
package main

import "github.com/mattn/go-gtk/gdk"

// Design tokens — single source of truth for all colors, sizes, and fonts.
const (
	C_BG        = "#1C1C1C"
	C_SURFACE   = "#2A2A2A"
	C_INPUT     = "#303030"
	C_TEXT      = "#D2D2D2"
	C_WHITE     = "#FFFFFF"
	C_MUTED     = "#9B9B9B"
	C_ACCENT    = "#5DC2AF"
	C_CB_BORDER = "#D9D9D9"
)

func color(hex string) *gdk.Color {
	return gdk.NewColor(hex)
}